core/src/main/java/tc/oc/pgm/controlpoint/ControlPoint.java

```diff
@@ -404,7 +407,7 @@ private void dominateAndFireEvents(@Nullable Competitor dominantTeam, Duration d
    * <p>If there is no neutral state, then the point is always either being captured by a specific
    * team, or not being captured at all.
    */
-  private void dominate(Competitor dominantTeam, Duration dominantTime) {
+  private void dominate(Competitor dominantTeam, Duration dominantTime, boolean contested) {
```

Comment:
I'd introduce an illegal argument exception thrown if dominant team is not null and contested is true, as that should never happen. I'd also double-check that it is in fact never happening